asm/citadel-patch/citadel-patch.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
```

Comment:
We need discuss more about this asm/citadel-patch/citadel-patch.yaml, it will be detected by anthoscli and generate error like this:

```
Unexpected error: error reconciling objects: error reconciling IstioOperator:/default-istio-operator: gke.io/cluster is required for IstioOperator:/default-istio-operator
```

Reply:
I moved the citadel-patch out of asm and asm-patch, should fix this now.